common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -4895,7 +4895,7 @@ public static enum ConfVars {
         "(size/arena_count) bytes. This size must be <= 1Gb and >= max allocation; if it is\n" +
         "not the case, an adjusted size will be used. Using powers of 2 is recommended."),
     LLAP_IO_MEMORY_MAX_SIZE("hive.llap.io.memory.size", "1Gb", new SizeValidator(),
-        "Maximum size for IO allocator or ORC low-level cache.", "hive.llap.io.cache.orc.size"),
+        "Maximum size for IO allocator or ORC low-level cache."),
```

Comment:
why should we remove the altName for this config? i think that was kept for backward compatibility

Reply:
Make sense. Reverted it.